### DISCUSSION TITLE
GSC ubuntu package or just clone for github?

### DISCUSSION THREAD

**Discussion by nmwael** [gramineproject/gramine#1239] [General]
Hi I've been working a bit with gramine, and wanted to see how the GSC project was, how am I supposed to install GSC?

Should I just clone the github repo or are there a more official way? I am on ubuntu and can see no packages in the gramine repo concerning gsc?

**Comment by mkow** [comment]
Hi,

It's just a bunch of Python scripts, you don't need to install anything, just clone the repo.

**Reply by nmwael** [reply]
Ok that was my thought aswell:)
